homeassistant/components/synology_dsm/backup.py

```diff
@@ -95,6 +106,19 @@ def _file_station(self) -> SynoFileStation:
             assert self.api.file_station
         return self.api.file_station
 
+    async def _async_suggested_filenames(
+        self,
+        backup_id: str,
+    ) -> tuple[str, str]:
+        """Suggest filenames for the backup.
+
+        :param backup_id: The ID of the backup that was returned in async_list_backups.
+        :return: A tuple of tar_filename and meta_filename
+        """
+        if (backup := await self.async_get_backup(backup_id)) is None:
+            raise BackupAgentError("Backup not found")
+        return suggested_filenames(backup)
```

Comment:
```suggestion
	filename = suggested_filename(backup)
        return (filename, f"{filename}_meta.json")
```

`from homeassistant.components.backup import suggested_filename`

Reply:
this would result in `("blabla.tar", "blabla.tar_meta.json")`